joomla/joomla-cms
# [4.3] php-diff line endings

Pull Request for Issue #40065 .

### Summary of Changes
update php-diff from @jfcherng to force the diff to ignore line ending differences


### Testing Instructions
@RickR2H please see if this solves your problem. I can't test it myself as I can't replicate the reported problem. But the logic of the change makles sense


### Actual result BEFORE applying this Pull Request



### Expected result AFTER applying this Pull Request



### Link to documentations
Please select:
